pkgs/development/compilers/gnu-smalltalk/default.nix

```
@@ -25,6 +25,8 @@ in stdenv.mkDerivation rec {
     sha256 = "1k2ssrapfzhngc7bg1zrnd9n2vyxp9c9m70byvsma6wapbvib6l1";
   };
 
+  patches = [ ./fix_mkorder.patch ];
```

```suggestion
  patches = [
     # <Explain why this patch is needed>
    ./0000-fix_mkorder.patch
  ];
```

Since patches aren't commutative (except maybe in Darcs), you should number them.